### PR TITLE
Factor out latex details

### PR DESCRIPTION
**Pull request by tobiasgrosser**

### CODE REVIEW

File: grammarly.tex [hunk 1]
```diff
@@ -1,4 +0,0 @@
-\def\grammarly{on}
```
Comment (math-fehr) [inline grammarly.tex:1]: Should we remove it from the `Makefile` as well then?

**Review by math-fehr — APPROVED** [review]
Yes, I like splitting this!

File: grammarly.tex (hunk 1 above)
Comment (tobiasgrosser) [reply grammarly.tex:1]: This was already removed in 97cc4b747d7160c7585cd651baf7cb7d6a96da09/

Comment (math-fehr) [reply grammarly.tex:1]: Oh okay, then that's good!

**Review by webmiche — APPROVED** [review]
Nice this definitely goes in the right direction!

**Pull request merged**